src/DetailsView/components/adhoc-issues-test-view.tsx

```diff
@@ -43,6 +43,7 @@ export type AdhocIssuesTestViewProps = {
     instancesSection: ReactFCWithDisplayName<CommonInstancesSectionProps>;
     narrowModeStatus: NarrowModeStatus;
     selectedVisualizationType: VisualizationType;
+    includeStepsText?: boolean;
```

Comment:
why we need ? , is this optional type?

Reply:
added the property includeStepsText?: boolean; in AssessmentIssuesTestViewProps instead of AdhocIssuesTestViewProps.